modules/creating-infra-machines-from-compute.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+//
+// * machine_management/creating-infrastructure-machinesets.adoc
+
+[id="creating-infra-machines-from-compute_{context}"]
+= Creating infrastructure machines
+
+You can create infrastructure machines by applying a custom label to an existing compute machine.
+
+.Prerequisites
+
+* You have created a compute machine set that has provisioned compute machines.
+
+.Procedure
+
+. List the compute nodes available in the cluster:
++
+[source,terminal]
+----
+$ oc get nodes -l node-role.kubernetes.io/worker
+----
++
+.Example output
+[source,terminal]
++
+----
+NAME                           STATUS   ROLES    AGE   VERSION
+ip-10-0-205-158.ec2.internal   Ready    worker   64m   v1.20.0+551f7b2.20.0
+ip-10-0-170-223.ec2.internal   Ready    worker   75m   v1.20.0+551f7b2.20.0
+ip-10-0-211-16.ec2.internal    Ready    worker   75m   v1.20.0+551f7b2.20.0
+ip-10-0-179-95.ec2.internal    Ready    worker   75m   v1.20.0+551f7b2.20.0
+----
+
+. Remove the `worker` label from the node you want to use as an infra node:
++
+[source,terminal]
+----
+$ oc label node <node_name> node-role.kubernetes.io/worker-
+----
+
+. Add the `infra` label to the node:
++
+[source,terminal]
+----
+$ oc label node <node_name> node-role.kubernetes.io/infra=""
+----
+
+[IMPORTANT]
+====
+If you are creating an infra machine for the first time, make sure there is a dedicated machine config pool. This is required to ensure the Machine Config Operator (MCO) can manage your infra machines.
```

Comment:
Same as above, link to the procedure on how to create a dedicated machine config pool for infra nodes

Reply:
In the assembly, the module for creating a dedicated MCP is directly after this note. I've tweaked the text to make it sound less like a prereq and more like a "do this next, if necessary".